### ISSUE TITLE
[Firefox OS Simulator v2.0, v2.1, v2.2, v3.0] No connection to Twitter

### ISSUE THREAD

**Issue by bobcanthelpyou** [minego/macaw-enyo#65]
Can't connect Macaw to Twitter (after the 'Select an account type' dialog) on Firefox OS 2.0, 2.1, 2.2 and 3.0; but it does work on Firefox OS 1.3 and 1.4. 

There are no errors on the web console of WebIDE, all GET and POST request response with HTTP/1.1 200 OK. But the 'csp_report' POST request seems different to the requests on Firefox 1.3 and 1.4
![webide](https://cloud.githubusercontent.com/assets/4523264/7059567/4891bace-de73-11e4-9938-3d0e2bce70f2.png)

Firefox OS 2.0
![firefoxossimulator20](https://cloud.githubusercontent.com/assets/4523264/7059374/3b26d8cc-de70-11e4-9845-dae670bca653.png)

Firefox OS 2.1
![firefoxossimulator21](https://cloud.githubusercontent.com/assets/4523264/7059371/32ef5dd2-de70-11e4-8892-0b3bedaa790d.png)

master branch Macaw / Firefox OS Simulator 2.0, 2.1, 2.2, 3.0

**Comment by minami-o** [comment]
Mmm isn't there something wrong with your setup?
I'm a Macaw user on FFOS 3.0 and it's still working perfectly well.
I just tried to add a new account, and no problem either, the twitter authentication page appeared as expected.

**Comment by bobcanthelpyou** [comment]
I am not sure what could be wrong, my copy from GitHub is running smoothly on Firefox OS 1.3 and 1.4, but not on higher versions of Firefox OS. But i am only tested it with Firefox OS Simulator, not on my Firefox OS 2.0 device, yet. Maybe something is wrong with the Simulators on Firefox 27.0.1?

**Comment by juaneladio** [comment]
I can confirm that it doesn't work on Firefox OS 2.0 (Keon). It just appears a black window after I choose the Twitter autjentication.

Months ago I had the same problem in a Firefox OS tablet but I though it was a specific problen with the tablet (running FxOS 2.0 too)

**Comment by bobcanthelpyou** [comment]
@juaneladio you used the origin/master branch of Macaw? Do you see something wrong in the WebIDE console log?

I installed Macaw successfully on my Firefox OS 2.0 device, today. I had no problems with the Twitter authentication.

**Comment by juaneladio** [comment]
First of all, I tested the current Marketplace version, not the origin/master branch. I encourage @minego to update that version too.

@bobcanthelpyou When I followed the deploy instructions of this repository I noticed the error in WebIDE. I attach screenshots, but I believe is a problem with the Firefox OS Simulator and the Content Security Policy in Twitter.

![captura de pantalla de 2015-05-04 22 31 03](https://cloud.githubusercontent.com/assets/547722/7466757/3323d17c-f2af-11e4-8a94-a9fa198908b5.png)

![captura de pantalla de 2015-05-04 22 31 38](https://cloud.githubusercontent.com/assets/547722/7466765/5419ad34-f2af-11e4-9a11-7d3832e5d59e.png)

![captura de pantalla de 2015-05-04 22 31 43](https://cloud.githubusercontent.com/assets/547722/7466769/60aa14ee-f2af-11e4-9db4-df0f9ceb0008.png)

When I tested in my Keon device I don't have a POST request to twitter.com/i/csp-report and the app works correctly.

![captura de pantalla de 2015-05-04 22 35 03](https://cloud.githubusercontent.com/assets/547722/7466793/e5d0b650-f2af-11e4-997a-25a97caf4824.png)
